src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/config/PduHeadProperties.kt

```diff
@@ -0,0 +1,17 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.config
+
+import org.springframework.boot.context.properties.ConfigurationProperties
+import org.springframework.boot.context.properties.ConstructorBinding
+
+@ConstructorBinding
+@ConfigurationProperties(prefix = "pdu")
+data class PduHeadProperties(
+  val contacts: Map<String, EmailConfig> = mapOf()
```

Comment:
Confused me for a bit how this map gets populated, got it now. But its not a viable solution to hard code the email for each PDU head long term

Reply:
More spring magic on collections of properties.